trestle/core/commands/author/folders.py

```diff
@@ -214,6 +214,9 @@ def _measure_template_folder(
             instance_file_names.append(instance_file_name)
             if instance_file.suffix == '.md':
                 md_api = MarkdownAPI()
+                header, _ = md_api.processor.read_markdown_wo_processing(instance_file)
+                if header == {}:
+                    continue
```

Comment:
So I have a question here. This will skip over everything for this file. Is this intended? 

I would have thought if expected a header this should trigger a failure.

Reply:
that was the question I had, if we provide the flag `-header-validate` to the `folders` and instance contains no header shall it fail? Another question would be whether when validating a task using `folders` it should recursively check all instances or just instances in the root?

Reply:
From my perspective if we provide the flag and there is no header it should fail.

Reply:
Okay I think the answer here is it should validate against *any* template provided but I agree.

Reply:
Okay, I will come up with a more sophisticated approach and do the changes

Reply:
I have ignored subfolders in `folders` tasks for now as this what caused the issue in governed repo validation when updating to new template version. This small fix will ignore subfolders for now. Handling empty headers will require more work, I will create a separate issue for it.